optaweb-vehicle-routing-backend/src/test/java/org/optaweb/vehiclerouting/plugin/planner/RouteOptimizerImplTest.java

```diff
@@ -253,6 +254,44 @@ void removing_depot_impossible_when_there_are_other_locations() {
                 .withMessageContaining("depot");
     }
 
+    @Test
+    void when_depot_is_added_all_vehicles_should_be_moved_to_it() {
```

Comment:
Are multiple depots supported? Just be reading a name of the test I wonder what should be the behaviour when a second depot is added.

Reply:
No. Just a single depot at the moment.